tests/SideBySide/QueryTests.cs

```diff
@@ -795,6 +795,32 @@ public void ReturnDerivedTypes()
 			}
 		}
 
+#if !NETCOREAPP1_1_2
+		[Fact]
+		public void GetSchemaTable()
+		{
+			var csb = AppConfig.CreateConnectionStringBuilder();
+			using (var connection = new MySqlConnection(csb.ConnectionString))
+			{
+				connection.Open();
+				using (MySqlCommand command = connection.CreateCommand())
+				{
+					command.CommandText = "select Host, ssl_cipher, max_connections, plugin, authentication_string from mysql.user limit 1";
```

Comment:
If this were moved to DataTypesTests, it could test everything set up in `DataTypesFixture` (https://github.com/mysql-net/MySqlConnector/blob/master/tests/SideBySide/DataTypesFixture.cs), which should provide very broad coverage of all the data types.

Reply:
Fixed.